virttest/utils_misc.py

```diff
@@ -477,8 +478,7 @@ def log_line(filename, line):
                 pass
             _open_log_files[base_file] = open(log_file, "w")
         timestr = time.strftime("%Y-%m-%d %H:%M:%S")
-        line = unicode("%s: %s" % (timestr, line), errors="ignore")
-        line = line.encode(ENCODING, errors='ignore')
+        line = string_safe_encode(line)
```

Comment:
Oups, I'm sorry about the missing time, fix in the next version along with removing the unnecessary ENCODING line.